e2e/protocol.test.js

```diff
@@ -269,6 +269,22 @@ describe('executeScript', () => {
     })
 })
 
+it('can handle navigation and command execution', async () => {
+    await browser.navigateTo('http://guinea-pig.webdriver.io')
+    expect(typeof (await browser.getTitle())).toContain('string')
```

Comment:
why it is not `'WebdriverJS Testpage'` ?

Reply:
see comment below